armi/reactor/reactorParameters.py

```diff
@@ -309,12 +328,14 @@ def defineCoreParameters():
 
         pb.defParam(
             "THmaxDeltaPPump",
-            units="Pa",
+            units=units.PASCALS,
             description="The maximum pumping pressure rise required to pump the given mass flow rate through the rod bundle",
         )
 
         pb.defParam(
-            "THmaxDilationPressure", units="", description="THmaxDilationPressure"
+            "THmaxDilationPressure",
+            units=units.UNITLESS,
```

Comment:
I would assume that this should actually have units of Pascal?

Reply:
Okay, yeah, I was able to confirm this has the same units as `THdilationPressure`, which are Pascals.

Sorry, I didn't not look carefully enough for units in downstream repos for clues.